packages/ckeditor5-dev-tests/tests/tasks/runautomatedtests.js

```diff
@@ -122,14 +124,14 @@ describe( 'runAutomatedTests', () => {
 
 				expect( stubs.fs.writeFileSync.calledOnce ).to.equal( true );
 				expect( stubs.fs.writeFileSync.firstCall.args[ 0 ] ).to.equal( '/workspace/build/.automated-tests/entry-point.js' );
-				expect( stubs.fs.writeFileSync.firstCall.args[ 1 ] ).to.equal( expectedEntryPointContent );
+				expect( stubs.fs.writeFileSync.firstCall.args[ 1 ] ).to.include( expectedEntryPointContent );
```

Comment:
Let's keep these tests strict, making sure that `expectedEntryPointContent` contains also the code generated for handling `console.*` members, will prevent us from accidentally dropping this feature.